ROADMAP.md

```diff
@@ -8,32 +8,32 @@ We hope that the items listed below will inspire further engagement from the com
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.11
+## Rook Ceph 1.12
 
-The following high level features are targeted for Rook v1.11 (January 2023). For more detailed project tracking see the [v1.11 board](https://github.com/rook/rook/projects/27).
+The following high level features are targeted for Rook v1.12 (July 2023). For more detailed project tracking see the [v1.12 board](https://github.com/rook/rook/projects/29).
 
-* Use more specific cephx accounts to better differentiate the source of Ceph configuration changes [#10169](https://github.com/rook/rook/issues/10169)
 * Enable restoring a cluster from the OSDs after all mons are lost [#7049](https://github.com/rook/rook/issues/7049)
 * Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
-* Support RBD mirroring across clusters with overlapping networks [#11070](https://github.com/rook/rook/issues/11070)
 * OSD encryption on partitions [#10984](https://github.com/rook/rook/issues/10984)
 * Object Store
+  * Support RGW LDAP integration [#4315](https://github.com/rook/rook/issues/4315)
   * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
   * Add alpha support for COSI (Container object storage interface) with K8s 1.25 [#7843](https://github.com/rook/rook/issues/7843)
-  * Support the immutable object cache [#11162](https://github.com/rook/rook/issues/11162)
-* Ceph-CSI v3.8
-  * CephFS encryption support [#3460](https://github.com/ceph/ceph-csi/pull/3460)
-* Update the operator sdk for internal CSV generation [#10141](https://github.com/rook/rook/issues/10141)
-* [Krew plugin](https://github.com/rook/kubectl-rook-ceph) features planned in the 1.11 time frame
+  * Pool sharing for clusters where many object stores are required [#11411](https://github.com/rook/rook/issues/11411)
+* Ceph-CSI v3.9
+
+## Krew Plugin
+
+Features planned in the 1.12 time frame for the [Krew Plugin](https://github.com/rook/kubectl-rook-ceph)
+  * Release a rewrite of the plugin in golang to lay the groundwork for future features [#76](https://github.com/rook/kubectl-rook-ceph/issues/76)
+  * Easy command for deploying a test cluster [#95](https://github.com/rook/kubectl-rook-ceph/issues/95)
   * Recover the CephCluster CR after accidental deletion [#68](https://github.com/rook/kubectl-rook-ceph/issues/68)
   * Collect details to help troubleshoot the csi driver [#69](https://github.com/rook/kubectl-rook-ceph/issues/69)
-  * Restore mon qourum from OSDs after all mons are lost [#7049](https://github.com/rook/rook/issues/7049)
 
 ## Themes
 
 The general areas for improvements include the following, though may not be committed to a release.
 
-* OSD encryption key rotation [#7925](https://github.com/rook/rook/issues/7925)
```

Comment:
Hey @travisn, do you think it's a good idea to take out the `Support for Windows nodes` feature from Themes? We've stopped working on it and we're confident that it won't be included in the upcoming 1.12 release.

Reply:
Yes, but it was started around two years back and is now a stale item for CSI. It might come/might not come to cephCSI.

Reply:
lets keep it for now and see if we need to remove it later.

Reply:
Right, it's still a future wish, so we can keep it in this section for now